modules/post/multi/manage/local_mitm.rb

```diff
@@ -0,0 +1,58 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::Priv
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'          => 'Local MITM',
+      'Description'   => %q{
+        This module setup a local proxy on the victim computer so you can MITM it.
+      },
+      'License'       => MSF_LICENSE,
+      'Author'        => [ 'Eliott Teissonniere' ],
+      'Platform'      => [ 'win', 'linux' ],
+      'SessionTypes'  => [ 'meterpreter', 'shell' ]
+    ))
+
+    register_options(
+      [
+        OptString.new('RHOST', [true, 'Host of the socks proxy server']),
```

Comment:
Why are these strings?

Reply:
It represents a way to choose the address of a remote proxy.
Maybe it is better to rename that has `PROXYHOST` and `PROXYPORT`?

Reply:
I think they should be `OptAddress` and `OptPort` or using the convenience methods `Opt::RHOST` and `Opt::RPORT`.

Reply:
Ah, so I wasn't obliged to do that... Ok! Didn't know any of these helpers